### PR TITLE
Remove Build Warnings

### PR DESCRIPTION
**Pull request by thefringeninja**

### CODE REVIEW

**Comment by thefringeninja** [comment]
The only other warning I see on all platforms now is
```
warning MSB4121: The project configuration for project "EventStore.ClientAPI.v5.Tests" was not specified in the solution file for the solution configuration "Release|Any CPU"
```

File: src/EventStore.ClientAPI/CatchUpSubscriptionFilteredSettings.cs [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
 		public CatchUpSubscriptionFilteredSettings(int maxLiveQueueSize, int readBatchSize, bool verboseLogging,
-			bool resolveLinkTos, int maxSearchWindow, string subscriptionName = "") : base(maxLiveQueueSize, readBatchSize, verboseLogging,
+			bool resolveLinkTos, int maxSearchWindow, string subscriptionName = "") : base(maxLiveQueueSize,
+			readBatchSize, verboseLogging,
 			resolveLinkTos, subscriptionName) {
 			MaxSearchWindow = maxSearchWindow;
 		}
 
+		/// <summary>
+		///
```
Comment (jageall) [inline src/EventStore.ClientAPI/CatchUpSubscriptionFilteredSettings.cs]: We should add an actual summary for the 2 remaining xml summary comments

File: src/EventStore.ClientAPI/CatchUpSubscriptionFilteredSettings.cs [hunk 2]
```diff
@@ -1,23 +1,40 @@
 using System;
 
-namespace EventStore.ClientAPI
-{
+namespace EventStore.ClientAPI {
+	/// <inheritdoc />
 	public class CatchUpSubscriptionFilteredSettings : CatchUpSubscriptionSettings {
+		/// <summary>
+		///
```
Comment (jageall) [inline src/EventStore.ClientAPI/CatchUpSubscriptionFilteredSettings.cs]: We should add an actual summary for the 2 remaining xml summary comments

File: src/EventStore.Transport.Tcp/TcpConnectionSsl.cs [hunk 3]
```diff
@@ -126,7 +126,7 @@ public class TcpConnectionSsl : TcpConnectionBase, ITcpConnection {
 
 				_sslStream = new SslStream(new NetworkStream(socket, true), false);
 				try {
-					var enabledSslProtocols = SslProtocols.Tls12 | SslProtocols.Tls11 | SslProtocols.Default;
+					var enabledSslProtocols = SslProtocols.Tls12 | SslProtocols.Tls11 | SslProtocols.Tls13;
```
Comment (pgermishuys) [inline src/EventStore.Transport.Tcp/TcpConnectionSsl.cs]: Please see https://github.com/EventStore/EventStore/pull/2068#issuecomment-561067177.

File: src/EventStore.sln [hunk 4]
```diff
@@ -376,6 +376,10 @@ Global
 		{BFBACE2B-6474-4B8A-B335-3311BB98DDF4}.Release|x86.Build.0 = Release|Any CPU
 		{8B3FDD14-C959-4859-80C4-B157D1D94969}.Release|x86.ActiveCfg = Release|Any CPU
 		{8B3FDD14-C959-4859-80C4-B157D1D94969}.Release|x86.Build.0 = Release|Any CPU
+		{8B3FDD14-C959-4859-80C4-B157D1D94969}.Debug|x64.ActiveCfg = Debug|Any CPU
```
Comment (jageall) [inline src/EventStore.sln]: Where these missing configs generating warnings or is this a separate issue?

Comment (thefringeninja) [reply src/EventStore.sln]: Generating warnings

**Pull request merged**